pyorbital/tlefile.py

```diff
@@ -53,41 +53,58 @@
 PKG_CONFIG_DIR = os.path.join(os.path.realpath(os.path.dirname(__file__)), 'etc')
 
 
-def read_platform_numbers(in_upper=False, num_as_int=False):
+def _get_config_path():
+    """Get the config path for Pyorbital."""
+    if 'PPP_CONFIG_DIR' in os.environ and not os.getenv('PYORBITAL_CONFIG_PATH'):
+        LOGGER.warning(
+            'The use of PPP_CONFIG_DIR is not supported anymore, ' +
+            'please use PYORBITAL_CONFIG_PATH if you need a custom config path for pyorbital!')
```

Comment:
So no deprecation period for `PPP_CONFIG_DIR`?

Reply:
I think you should definitely have a version or two deprecation where setting PPP_CONFIG_DIR still works but issues a warning.

Reply:
Yeah, I fixed this now. Chose the next major version, 1.7, as the deadline. First I thought 1.8, perhaps that is preferable?

Reply:
What do you say @pnuu @mraspaud and @djhoese shoud we go for 1.7 or 1.8, the version that will remove the backward compatibilty for PPS-CONFIG_DIR?
Judging from the recent years it is not so often we bump the second digit:
```
* 2021-04-12 11:48:04 +0200 6b7e685  (tag: v1.6.1)
* 2021-03-09 17:44:47 +0100 af4cc3c  (upstream/master)
* 2021-03-08 18:06:23 +0100 12f2fae  (origin/fix-numpy-deprecation-warnings, fix-numpy-deprecation-warnings)
* 2020-06-24 13:56:04 +0200 3ff7a6b  (tag: v1.6.0, origin/master, origin/main, origin/HEAD, checkout)
* 2018-11-16 14:01:27 -0600 5c8ac6f  (tag: v1.5.0)
* 2018-10-23 11:29:26 +0200 eb3dadc  (tag: v1.4.0)
* 2018-04-09 16:41:48 +0200 5c1036e  (tag: v1.3.1)
```

Reply:
So last release was 1.7.3, right (you may have to update your local copy to see those tags)? So I think deprecating and releasing this in 1.8.0 and then removing it in 1.9.0 is OK.

Reply:
Fixed. Will stop working from 1.9.